README.md

```diff
@@ -1,38 +1,67 @@
 What is Notepad++ ?
 ===================
 
-[<img src="https://notepad-plus-plus.org/assets/images/communityDiscussionOrange.png">](https://notepad-plus-plus.org/community/)  &nbsp;&nbsp;&nbsp;&nbsp;[![Join the chat at https://gitter.im/notepad-plus-plus/notepad-plus-plus](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/notepad-plus-plus/notepad-plus-plus?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

```

Comment:
I think @donho changed the forum badge to an orange version after you updated this commit...
#354 should fix this and use the I made in SVG...
@donho needs to upload it to https://notepad-plus-plus.org/assets/images/ for permanent use, however.